client/components/login-window/docs/example.jsx

```diff
@@ -0,0 +1,17 @@
+import useLoginWindow from 'calypso/components/login-window';
+
+const LoginWindowExample = () => {
+	const { login } = useLoginWindow();
```

Comment:
This example usage is out of date right?

Reply:
Yeah it is, I'll update it when I've settled on a working usage